src/main/java/echochat/TaskList.java

```diff
@@ -0,0 +1,46 @@
+package echochat;
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private ArrayList<Task> lst = new ArrayList<>();
```

Comment:
this field can be renamed to taskList instead of just lst so it is easier to understand